src/components/racer-basics/index.js

```diff
@@ -47,6 +57,8 @@ class RacerBasics {
     this.$exampleData.setDiff(path, value)
   }
 
+  //// i would keep TODO's in a separate branch instead of having comments mixed in with master
```

Comment:
We have a common pattern for adding TODOs in our codebase for things that need local changes, so I think this'll be ok for now. I added a blurb to the TODO to make it more clear that this section is WIP.

Also, these TODOs should go away really soon in general as this tutorial matures, b/c these are basic features, and we can track bigger issues with Github.